code-of-conduct.md

```diff
@@ -70,6 +70,9 @@ Examples of unacceptable behavior include but are not limited to:
 * Stalking or following someone without their consent
 * Unwelcome physical contact
 * Unwelcome sexual or romantic attention or advances
+* Using CNCF projects or community spaces for political campaigning or  promotion of political causes 
```

Comment:
mostly nits!!
1. drop space before "promotion"
2. `For clarity, this does not restrict individuals' personal attire` could be phrased better. `To clarify, this policy does not restrict individuals from expressing personal views through their attire` perhaps?
3. Drop "Please see the Code of Conduct FAQ for more information." as the faq does NOT yet cover attire (and we don't need to add that to all the lines, there's is separate FAQ section on line 96)
4. Is there a PR for adding attire to FAQ already?

thanks @jeremyrickard !

Reply:
@dims I pushed a new commit that I think addresses comments 1-3. 

I don't believe we have added a PR for the FAQ change. Let me verify if we have that ready to go and I'll handle it.

Reply:
thanks @jeremyrickard !

LGTM